### PR TITLE
Change the template-wf-wizard.jsp file to skip the validation of the newly added workflow  mediator  template  form

### PR DESCRIPTION
**Pull request by malithiperera**
WSO2 currently utilizes a multi-step approval template for managing workflow processes. With the introduction of a new workflow mediator template, we aim to skip the complex validation process only when the new template is selected.

### CODE REVIEW

File: components/workflow-mgt/org.wso2.carbon.identity.workflow.mgt.ui/src/main/resources/web/workflow-mgt/template-wf-wizard.jsp [hunk 1]
```diff
@@ -239,7 +239,14 @@
 
         var stepOrder = 0;
         function nextWizard(){
-            if(!validateInputs()){
+        <%
+        boolean intTemp = true;
+        if(template !=null && template.getName().equals("External Workflow Template")){
```
Comment (sadilchamishka) [inline components/workflow-mgt/org.wso2.carbon.identity.workflow.mgt.ui/src/main/resources/web/workflow-mgt/template-wf-wizard.jsp:244]: Let's define `"External Workflow Template"` as a constant in the `WFConstant` class and use that constant here.

File: components/workflow-mgt/org.wso2.carbon.identity.workflow.mgt.ui/src/main/resources/web/workflow-mgt/template-wf-wizard.jsp [hunk 2]
```diff
@@ -239,7 +239,14 @@
...
+            intTemp = false;
+        }
+        %>
+            var abc = "<%=intTemp%>";
```
Comment (sadilchamishka) [inline components/workflow-mgt/org.wso2.carbon.identity.workflow.mgt.ui/src/main/resources/web/workflow-mgt/template-wf-wizard.jsp:248]: Let's try to simplify the logic of using java variables within javascript code

Comment (sadilchamishka) [reply components/workflow-mgt/org.wso2.carbon.identity.workflow.mgt.ui/src/main/resources/web/workflow-mgt/template-wf-wizard.jsp:248]: Ref - https://github.com/wso2-support/identity-apps/pull/319/files#diff-bffbd72fca2790d1abafde5ff9360318c6859cbcce523989782f98e389150aaeR92

Comment (malithiperera) [inline components/workflow-mgt/org.wso2.carbon.identity.workflow.mgt.ui/src/main/resources/web/workflow-mgt/template-wf-wizard.jsp:248]: write the purpose and change the variable names

Comment (malithiperera) [reply components/workflow-mgt/org.wso2.carbon.identity.workflow.mgt.ui/src/main/resources/web/workflow-mgt/template-wf-wizard.jsp]: will be fix in next PR

**Pull request merged**